subprojects/docs/src/docs/userguide/javaPlugin.xml

```diff
@@ -440,7 +472,7 @@
         <figure>
             <title>Java plugin - dependency configurations</title>
             <imageobject>
-                <imagedata fileref="img/javaPluginConfigurations.png"/>
+                <imagedata fileref="img/java-library-ignore-deprecated-main.png"/>
```

Comment:
We can't reuse the `java-library` graph 1:1 here, because it contains the `api` configuration, which the `java` plugin doesn't have.